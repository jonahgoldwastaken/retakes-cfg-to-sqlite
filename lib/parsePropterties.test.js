const parseProperties = require('./parsePropterties')

test('parseProperties parses cfg json for sql database', () => {
  const testContent = [
    {
      origin: '32.151203 -908.485229 -85.425278',
      angle: '1.429899 -151.466568 0.000000',
      bombsite: 'B',
      team: 'CT',
      type: '0',
    },
  ]

  expect(parseProperties('de_inferno.cfg', testContent)).toMatchSnapshot(
    'parsedObjectReadyToSave'
  )
})
