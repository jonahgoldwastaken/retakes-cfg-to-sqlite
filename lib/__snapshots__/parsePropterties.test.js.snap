// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseProperties parses cfg json for sql database: parsedObjectReadyToSave 1`] = `
Array [
  Object {
    "angx": 1.429899,
    "map": "de_inferno",
    "posx": 32.151203,
    "posy": -908.485229,
    "posz": -85.425278,
    "site": 1,
    "type": 3,
  },
]
`;
